md_doc/pid.md

```diff
@@ -0,0 +1,15 @@
+Process id
+------------
+
+Can be used as an integer type by simple casting. For example:
+
+```
+use sysinfo::AsU32;
+
+let p: Pid = 0;
+let i: u32 = p.as_u32();
+```
+
+On glibc systems this is a glibc [`pid_t`](https://www.gnu.org/software/libc/manual/html_node/Process-Identification.html).
```

Comment:
I'm not sure it's very relevant but I guess it doesn't hurt to have it so thanks!